pages/game/roleScreen.tsx

```diff
@@ -8,16 +8,28 @@ interface RoleProps {
 }
 
 const RoleScreen: React.FC<RoleProps> = ({ data }) => {
-  const userId = useUserId();
 
   if (!data) {
     return (<></>);
   }
 
+  const unveilRole = ()=>{
+    const roleCard = document.querySelector(".roleCard")
+    console.log("REVEALED", roleCard, data)
+    if(roleCard && data.url){
+      roleCard.style.backgroundimage = data.url
+    }
+  }
+
   return (
     <>
-      <div className='roleScreen'>
-        {data.ability}
+      <div className="roleCard" >
+        <h1 className="Role">{data.role}</h1>
+        <h3 className='Title'>{data.title}</h3>
+        <div className='ability'>
+          <p> {data.ability}</p>
```

Comment:
For readability/consistency we should change all of the react interpreters to have the same formatting. I tend to prefer `> {data.ability} <`

Reply:
fixed